operator/templates/config.yaml

```
@@ -0,0 +1,18 @@
+      healthProbeBindAddress: :8081
+    metrics:
+      bindAddress: 127.0.0.1:8080
+    webhook:
+      port: 9443
+    leaderElection:
+      leaderElect: true
+      resourceName: b418339d.teamdev.com
+kind: ConfigMap
+metadata:
+  name: operator-manager-config
+  namespace: {{.Values.namespace}}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)